src/commands/flinkArtifacts.ts

```diff
@@ -1,9 +1,25 @@
-import { Disposable } from "vscode";
+import vscode, { Disposable } from "vscode";
```

Comment:
Minor: let's either change this to `import * as vscode from "vscode"` or `import { Disposable, window, workspace } from "vscode"` instead of mixing imports